### PR TITLE
(12.3) Additional preview URLs support in Delivery API 

### PR DESCRIPTION
**Pull request by elit0451**
## Description
- Adding an article about configuring additional Preview URLs in the Delivery API, that will be possible from v12.3;
- Minor formatting fixes in other Delivery API articles - to be consistent.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [X] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
CMS v12.3

## Deadline (if relevant)
As soon as possible, but no rush. Somewhere between 19th Oct and the final 12.3 release on 2nd Nov.

### CODE REVIEW

**Review by kjac — APPROVED** [review]
This is excellent stuff, @elit0451 🥇 🔥

File: 12/umbraco-cms/reference/content-delivery-api/README.md [hunk 1]
```diff
@@ -723,3 +669,45 @@ public void ConfigureServices(IServiceCollection services)
...
+
+#### Grid Layout (legacy)
+
+The Legacy Grid in Umbraco is supported to a certain extent. However, keep in mind that it may not be suitable for headless content scenarios. Instead, we recommend using the Block Grid property editor.
+
+#### Rich Text Editor
+
+The Delivery API is not going to support the rendering of Macros within the Rich Text Editor. Therefore, any Macros included in the content will not be executed or output when retrieving content through the API.
+
+When outputting the Rich Text Editor content as HTML (_the default format_), be aware that internal links may be insufficient in a multi-site setup. There is a possibility that this limitation may be addressed in future updates. However, consider the alternative approach to rendering the RTE content as JSON.
```
Comment (alina-tincas) [inline 12/umbraco-cms/reference/content-delivery-api/README.md:699]:
```suggestion
When outputting the Rich Text Editor (RTE) content as HTML (_the default format_), be aware that internal links may be insufficient in a multi-site setup. There is a possibility that this limitation may be addressed in future updates. However, consider the alternative approach to rendering the RTE content as JSON.
```

**Review by alina-tincas — COMMENTED** [review]
resolving lint error

File: 12/umbraco-cms/reference/content-delivery-api/README.md [hunk 2]
```diff
@@ -723,3 +669,45 @@ public void ConfigureServices(IServiceCollection services)
...
+
+#### Grid Layout (legacy)
+
+The Legacy Grid in Umbraco is supported to a certain extent. However, keep in mind that it may not be suitable for headless content scenarios. Instead, we recommend using the Block Grid property editor.
+
+#### Rich Text Editor
+
+The Delivery API is not going to support the rendering of Macros within the Rich Text Editor. Therefore, any Macros included in the content will not be executed or output when retrieving content through the API.
+
+When outputting the Rich Text Editor (RTE) content as HTML (_the default format_), be aware that internal links may be insufficient in a multi-site setup. There is a possibility that this limitation may be addressed in future updates. However, consider the alternative approach to rendering the RTE content as JSON.
```
Comment (alina-tincas) [inline 12/umbraco-cms/reference/content-delivery-api/README.md:699]:
```suggestion
When outputting the Rich Text Editor content as HTML (_the default format_), be aware that internal links may be insufficient in a multi-site setup. There is a possibility that this limitation may be addressed in future updates. However, consider the alternative approach to rendering the Rich Text Editor content as JSON.
```

**Review by alina-tincas — COMMENTED** [review]
resolving lint errors

File: 12/umbraco-cms/reference/content-delivery-api/additional-preview-environments-support.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+With Umbraco, you can save and preview draft content before going live. The preview feature allows you to visualize how a page will look once it is published, directly from within the backoffice. This is also possible for the Content Delivery API data. You can extend the preview functionality in the backoffice by configuring external preview URLs for client libraries consuming the Content Delivery API.
+
+{% hint style="info" %}
+To get familiar with the preview functionality in the Delivery API, please refer to the [Preview concept](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api#preview) section.
+{% endhint %}
+
+{% hint style="info" %}
+The support for configuring additional preview environments in the Delivery API was introduced in version 12.3.
+{% endhint %}
+
```
Comment (alina-tincas) [inline 12/umbraco-cms/reference/content-delivery-api/additional-preview-environments-support.md:17]:
```suggestion
{% hint style="info" %}
The support for configuring additional preview environments in the Delivery API was introduced in version 12.3.
{% endhint %}
```

**Review by alina-tincas — COMMENTED** [review]
removing extra space

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Hi @elit0451 thank you for the PR 🙌 and thank you @kjac for reviewing #h5YR both!

All looks good so am merging it in 😊

**Pull request merged**